mage_ai/streaming/constants.py

```diff
@@ -28,6 +29,7 @@ class SinkType(str, Enum):
     GOOGLE_CLOUD_STORAGE = 'google_cloud_storage'
     INFLUXDB = 'influxdb'
     KAFKA = 'kafka'
+    NATS = 'nats'
```

Comment:
You have not added the support for data exporter. Hence, this needs to be removed.